src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -338,6 +338,21 @@ protected Uri ParseServerUrl(ConfigurationKeyValue kv)
 			return DefaultValues.ServerUri;
 		}
 
+		protected Uri ParseProxyServerUrl(ConfigurationKeyValue kv)
```

Comment:
I think we also need parser for username and password strings similar to `ParseServerCert` to check for null or empty strings.